### ISSUE TITLE
Loading JPEG image with ImageMagick becomes very slow in Julia 1.7

### ISSUE THREAD

**Issue by etibarg** [JuliaImages/Images.jl#992]
Hello,

I've noticed a huge slowdown during image loading this morning, after migrating to julia 1.7.0.

Before:
```julia
julia> @time rgb = load("00070__.jpg")
  5.381018 seconds (97 allocations: 1.268 GiB, 3.26% gc time)
10652×14204 Array{RGB{N0f8},2} with eltype RGB{N0f8}:
```
After:
```julia
julia> @time rgb = load("00070__.jpg")
 85.531723 seconds (1.82 G allocations: 62.142 GiB, 8.78% gc time, 0.00% compilation time)
10652×14204 Array{RGB{N0f8},2} with eltype RGB{N0f8}:
```

The image weights ~30Mo.
Clearly, there's something wrong with the allocation. But what/why?

```julia
julia> versioninfo()
Julia Version 1.7.0
Commit 3bf9d17731 (2021-11-30 12:12 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, tigerlake)
```

Thanks,

NB: It's the first time I'm posting an issue. Sorry if it's the wrong place. (Should it be in julia repo?)

**Comment by johnnychen94** [comment]
The last step now: https://github.com/JuliaRegistries/General/pull/53821

**Comment by johnnychen94** [comment]
Closing this as JpegTurbo.jl is now available in ImageIO v0.6.1
